.github/workflows/pull-request-rfc.yml

```diff
@@ -0,0 +1,98 @@
+name: RFC Pull Request
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
```

Comment:
In general this is a very long script 
Can we extract this into its own file and call it that way / reference it?